docs/usage.md

```diff
@@ -6,58 +6,80 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+nf-core/funcscan is a pipeline for efficient and parallelised screening of long nucleotide sequences such as contigs for possible natural products sequences. It targets antimicrobial peptides, antimicrobial resistance genes, and biosynthetic clusters.
 
```

Comment:
Good question... @jasmezz would have to answer that, but I'm pretty sure antisMASH does a lot of different things: [`secondary metabolite biosynthetic gene clusters` these are presumably not solely antimicrobial?](https://academic.oup.com/nar/article/39/suppl_2/W339/2507123)

"Microbial secondary metabolites offer great potential for the development of new medicines. They belong to a wide variety of chemical classes, and many of them have cholesterol-lowering, anti-tumor or antibiotic activities."

" It can rapidly detect all known classes of secondary metabolite biosynthesis gene clusters, provide detailed NRPS/PKS functional annotation, and predict the chemical structure of NRPS/PKS products with higher accuracy than existing methods."